packages/react/src/components/AnimatedHeader/components/AnimatedHeader/AnimatedHeader.tsx

```diff
@@ -165,7 +167,7 @@ const AnimatedHeader: React.FC<AnimatedHeaderProps> = ({
       // Run Looped Animation
       animation.addEventListener('complete', loop);
     }
-  }, []);
+  }, [animationContainer, headerAnimation, isReduced]);
```

Comment:
I believe we should add a cleanup function here to make sure there are no dangling eventListeners.
```suggestion
    return () => {
      animation?.removeEventListener('DOMLoaded', reducedMotion);
      animation?.removeEventListener('DOMLoaded', load);
      animation?.removeEventListener('complete', loop);
    } 
  }, [animationContainer, headerAnimation, isReduced]);
```